src/NewButton/icon-button.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {forwardRef} from 'react'
```

Comment:
Should this - and the other files that export default React components - be renamed in PascalCase? Or is this intentional?

Reply:
Ah I see! Will change up the file names in a followup PR 👍